### PR TITLE
Heroku Deployment with Redis/Travis Integration

### PR DESCRIPTION
**Pull request by kave**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "contributors": [
+    {
+      "name": "Emmanuel Apau",
+      "email": "etapau@gmail.com",
+      "url": "https://github.com/kave"
+    },
+    {
+      "name": "Sean Lewis",
+      "email": "",
+      "url": "https://github.com/madridsoccer5 "
```
Comment (kave) [inline package.json:18]: @madridsoccer5 hey after you merge this don't forget to put in your email in your next pull request

File: .buildpacks [hunk 2]
```diff
@@ -0,0 +1,2 @@
+https://github.com/heroku/heroku-buildpack-nodejs.git
+https://github.com/heroku/heroku-buildpack-python.git
```
Comment (kave) [inline .buildpacks:2]: had to use a custom heroku buildpack so it knows to install npm dependencies (bower) before running the application

File: face_it/settings/base.py [hunk 3]
```diff
@@ -249,11 +248,14 @@
 
 LOGIN_URL = '/login'
 
+
+redis_url = os.environ.get('REDISTOGO_URL', 'redis://127.0.0.1:6379/1')
+
```
Comment (kave) [inline face_it/settings/base.py:253]: redis integration configuration. Actually server is being hosted by RedisToGo.com

File: .travis.yml [hunk 4]
```diff
@@ -1,14 +1,15 @@
...
-after_success:
-  coveralls
\ No newline at end of file
+- pip install -r requirements.txt
+- pip install coveralls
+script: coverage run --source=face_it manage.py test
+after_success: coveralls
+deploy:
+  provider: heroku
+  api_key:
```
Comment (kave) [inline .travis.yml:11]: travis deployment integration

File: package.json (hunk 1 above)
Comment (m3brown) [inline package.json:18]: Hey, didn't I contribute too? :)

Comment (kave) [inline package.json:18]: @m3brown lol of course. what email should i use for your details?

**Pull request merged**